source/vibe/http/common.d

```diff
@@ -601,7 +601,7 @@ struct CookieValueMap {
 
 	inout(string)* opBinaryRight(string op)(string name) inout if(op == "in")
 	{
-		foreach(c; m_entries)
+		foreach(ref c; m_entries)
```

Comment:
This gives me an error about escaping local references in the newer DMD. I was working with git head so any fix that was there wasn't working